proxy/http_headers.go

```diff
@@ -58,7 +58,12 @@ func addHeaders(r *http.Request, cfg config.Proxy, stripPath string) error {
 	// http proxy which sets it.
 	ws := r.Header.Get("Upgrade") == "websocket"
 	if ws {
-		r.Header.Set("X-Forwarded-For", remoteIP)
+		targetHeader := []string{remoteIP}
+		sourceHeader := r.Header.Get("X-Forwarded-For")
+		if sourceHeader != "" {
+			targetHeader = append([]string{sourceHeader}, targetHeader...)
+		}
+		r.Header.Set("X-Forwarded-For", strings.Join(targetHeader, ","))
```

Comment:
Down here, these should be joined by 

    ", "

as it is done in the standdard library here:

https://github.com/golang/go/blob/master/src/net/http/httputil/reverseproxy.go#L303